.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: 🐛 Bug
+description: Create a bug report to help us improve
+labels: ['Bug']
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing to iTwinUI by creating an issue!
+        Make sure you first do a search for existing issues so that this is is not a duplicate.
+        Also visit our [FAQ](https://github.com/iTwin/iTwinUI/wiki/FAQ) and fork our [sandbox](https://stackblitz.io/github/iTwin/iTwinUI/tree/main/minimal-sandbox?file=/src/App.tsx)
+  - type: textarea
+    attributes:
+      label: Describe the bug (current behavior)
+      description: A concise description of what you're experiencing.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Reproduction
+      description: 'Add link to a minimal repro'
```

The sandbox link and instructions should go here.

```suggestion
      label: Link to minimal repro
      description: 'Create a minimal reproduction by forking [this sandbox](https://stackblitz.io/github/iTwin/iTwinUI/tree/main/minimal-sandbox?file=/src/App.tsx)'
```